.github/workflows/docker.yml

```diff
@@ -15,14 +15,21 @@ jobs:
       -
         name: Check out
         uses: actions/checkout@v3
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
       -
         name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
```

Comment:
```suggestion
        uses: docker/setup-buildx-action@v1
        id: buildx
```

Reply:
I have removed the output block, as the QEMU step has the output already.